cli/unipipe/commands/git.ts

```diff
@@ -0,0 +1,99 @@
+import { Command } from "../deps.ts";
+import { Repository } from "../repository.ts";
+
+interface GitOpts {
+  name: string;
+  email: string;
+  message: string;
+}
+
+export function registerGitCmd(program: Command) {
+  program
+    .command("git <cmd> [repo]")
```

Comment:
f: You should define the available options via the options function here. See other commands how options are defined there.